# Send a custom command
string command
string json
---
bool success
